docs/dependency_confusion.md

```diff
@@ -61,80 +61,78 @@ to verify that the content of all direct and indirect dependencies is what you e
-> Cachito does not require integrity values, but newer versions of npm will always include them. Check
-> if all your dependencies have an integrity value, update your lock file if not.
->
-> Do not try to use private registries. If you point npm to a private registry (or any registry other
-> than the official one) via [.npmrc](https://docs.npmjs.com/cli/v6/configuring-npm/npmrc), Cachito
-> will ignore it and look in the official registry anyway.
->
-> ## yarn
->
-> *TL;DR: same as npm, but the handling of unofficial registries is slightly less surprising. Still,
-> you probably should not use them.*
->
-> Yarn packages are identical to npm packages except that they use
-> [yarn.lock](https://classic.yarnpkg.com/en/docs/yarn-lock/) as the lock file. Everything that
-> applies to npm applies to yarn.
->
-> The one difference is the handling of unofficial registries. If you point yarn to an unofficial
-> registry via .npmrc or [.yarnrc](https://classic.yarnpkg.com/en/docs/yarnrc), this will be reflected
-> in the resolved url in the lock file. Cachito will see that the url does not point to the official
-> registry and will treat the dependency as a plain https dependency. If the url is accessible to
-> Cachito, it will download the dependency directly without relying on npm/yarn dependency resolution.
-> That does not necessarily make using unofficial registries a good idea. If the registry is private,
-> your build will either fail or leak internal package names.
->
-> ## RubyGems (Bundler)
->
-> *TL;DR: allowing `https://rubygems.org` as the only source should mitigate the issue for GEM dependencies*
->
-> Cachito parses `Gemfile.lock` which pins all dependencies to exact versions. Cachito allows GEM dependencies
-> to be fetched only from `https://rubygems.org`, otherwise raises an error. GIT dependencies are specified using
-> a repo URL and pinned to a commit hash.
->
-> Bundler doesn't verify checksums of dependencies yet, however, there's an effort to bring
-> [internal GitLab implementation](https://gitlab.com/gitlab-org/gitlab/-/merge_requests/92633)
-> to [the upstream](https://github.com/rubygems/rubygems/pull/5808).
+## pip
+
+*TL;DR: if your package is Cachito compliant, it is most likely safe. If you want to be sure, verify
+checksums.*
+
+Python packages can have various formats, which are specified across multiple PEPs. Pip supports
+most of them. Cachito support for pip, on the other hand, is quite simple. You must define all the
+direct and indirect dependencies of your package in
+[requirements.txt](https://pip.pypa.io/en/stable/user_guide/#requirements-files) style files and
+tell Cachito to process them.
+
+Cachito further restricts what you can put in your requirements.txt files. All of the dependencies
+must be
+[pinned](https://github.com/release-engineering/cachito/blob/master/docs/pip.md#pinning-versions)
+to an exact version. Cachito will refuse to process requirements files that use the --index-url or
```

Comment:
cachi2 does support `--index-url` since quite recently, not sure about `--extra-index-url` from the top of my head, so it would be worth double checking. It may or may not have an impact on the rest of the paragraph content regarding the security implications.

Needless to say that ^these fixups should go to a standalone commit to keep the changes straightforward and easy to review.